dockers/docker-fpm-frr/critical_processes

```diff
@@ -0,0 +1,5 @@
+zebra
```

Comment:
Instead of restarting the whole docker, shouldn't zebra be restarted alone? Fpmsyncd can start the reconciliation with app-db when it reconnects to zebra.